src/pybind/mgr/dashboard/frontend/src/app/app.component.ts

```diff
@@ -8,7 +8,7 @@ import { AuthStorageService } from './shared/services/auth-storage.service';
 @Component({
   selector: 'cd-root',
   templateUrl: './app.component.html',
-  styleUrls: ['./app.component.scss']
+  styleUrls: ['./app.component.scss'],
```

Comment:
For me, it's ok to go with trailing commas
but Prettier:
npm run makePretty
erases them... What should we do about it?

Reply:
This comma was added just to trigger the prettier validation and test it in jenkins. It will be removed before the PR is merged.

I personally think its cleaner to not use trailing commas and we can use the same style for TS and JSON arrays/objects. 

Currently prettier is configured to remove all trailing commas. We can discuss a change to this option in the future, but changing it is not in the context of this PR.

Reply:
@alfonsomthd For now I would suggest to remove it.